### DISCUSSION TITLE
如果您的小程序项目使用了ColorUI GA，欢迎在这里分享。

### DISCUSSION THREAD

**Discussion by XiaokangLei** [XiaokangLei/ColorUI-GA#9] [Announcements]
如果您的小程序项目使用了ColorUI GA组件库，欢迎在这里分享您的小程序码。

我们需要一些优质的演示项目，也顺便能让更多人看见您的项目。

我们计划在仓库文档里，添加优质的小程序码展示。

关于如何定义优质？？
1.整体基于ColorUI GA组件的库设计开发。
2.漂亮、布局合理、颜色搭配合理。
3.非低俗，引流项目。
4.商业或免费都可以。

有时间限制吗？
没有，随时可以提交您的优质小程序码，我们会认真看完，并在团队内部进行优质打分，如果达到优质标准，就会放在仓库文档内展示。
不过，现在前期容易上榜，后期多了后，可能标准会变动。

**Comment by 1eez** [comment]
我的小程序使用了ColorUI-GA，目前注册用户5w（截止2024.4），月活6000左右。
原因是我自己是痛风患者，所以经常需要查询食材的嘌呤含量。
小程序码如下：
![小程序码3](https://github.com/XiaokangLei/ColorUI-GA/assets/17718224/f8fc6b13-b2aa-4920-b017-fb41f676fc2f)

**Comment by XiaokangLei** [comment]
MemoTime倒计时、备忘录（日记）、目标管理、积分签到小程序
![gh_3310ea29ed2f_258](https://github.com/XiaokangLei/ColorUI-GA/assets/19248640/bd3a1bf3-e2af-4b37-814e-200b71dbba13)

**Comment by flymaAqi** [comment]
基于colorUI GA2和ColorUI原版的模板，使用vue3+uniapp重构的小程序，平时喜欢积累一些好玩有趣的css样式，所以会持续更新该小程序，另外也因为uniapp支持跨平台的特性上架了安卓App，希望列为喜欢。
![flymaUI](https://github.com/user-attachments/assets/956f0f14-2fae-45f5-a904-3a095b203dc5)
![anzhuo](https://github.com/user-attachments/assets/c8e09aec-14bb-4ff6-b903-af62beaf6b09)
